### PR TITLE
Update Batch overview

### PR DESCRIPTION
**Pull request by JeremyLikness**
Added examples, links, overviews and samples. Tagging @CamSoper and will give to team for review and sign-off.

### CODE REVIEW

**Comment by openpublishbuild** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api/overview/azure/batch.md](https://github.com/JeremyLikness/azure-docs-sdk-dotnet/blob/patch-5/api/overview/azure/batch.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/overview/azure/batch?branch=pr-en-us-276) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C31%5C14458122-e42f-10e3-bd3d-ee1c7494cfbf%5CPullRequest%5C201707311412591442-276%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api/overview/azure/batch.md [hunk 1]
```diff
@@ -29,12 +37,44 @@ Install-Package Azure.Batch
 dotnet add package Azure.Batch
```
Comment (cecilphillip) [inline api/overview/azure/batch.md]: Why do we label the .NET CLI sections as bash? The command is the same for non-bash environments.

File: api/overview/azure/batch.md [hunk 2]
```diff
@@ -29,12 +37,44 @@ Install-Package Azure.Batch
...
+### Example
+
+The following example uses the client SDK to create a job to run in Azure Batch.
+
+```csharp
+/*
+using Microsoft.Azure.Batch.Auth;
+using Microsoft.Azure.Batch.Common;
+*/
+using (BatchClient batchClient = await BatchClient.OpenAsync(credentials))
```
Comment (cecilphillip) [inline api/overview/azure/batch.md]: Adding a line showing the creation of BatchSharedKeyCredentials, used in OpenAsync, might be helpful

File: api/overview/azure/batch.md [hunk 3]
```diff
@@ -29,12 +37,44 @@ Install-Package Azure.Batch
...
 ```
 
+### Example
+
+The following example uses the client SDK to create a job to run in Azure Batch.
+
+```csharp
+/*
+using Microsoft.Azure.Batch.Auth;
+using Microsoft.Azure.Batch.Common;
```
Comment (cecilphillip) [inline api/overview/azure/batch.md]: For the sample we only need the Microsoft.Azure.Batch namespace.
Microsoft.Azure.Batch.Auth if you add BatchSharedKeyCredentials. See comment below.

File: api/overview/azure/batch.md [hunk 4]
```diff
@@ -47,10 +87,40 @@ Install-Package Microsoft.Azure.Management.Batch.Fluent
...
 ```
 
+### Example
+
+The following example retrieves the quota for the subscription, creates an account, and regenerates the primary account key.
+
+```csharp
+/*
+using Microsoft.Azure.Management.Batch;
+using Microsoft.Azure.Management.Batch.Models;
```
Comment (cecilphillip) [inline api/overview/azure/batch.md]: Need to update to the fluent namespaces.

using Microsoft.Azure.Management.Batch.Fluent;
using Microsoft.Azure.Management.Batch.Fluent.Models;
using Microsoft.Rest;

File: api/overview/azure/batch.md [hunk 5]
```diff
@@ -47,10 +87,40 @@ Install-Package Microsoft.Azure.Management.Batch.Fluent
...
+*/
+using (BatchManagementClient batchManagementClient = new BatchManagementClient(new TokenCredentials(accessToken)))
+{
+    batchManagementClient.SubscriptionId = subscriptionId;
+
+    // Get the account quota for the subscription
+    BatchLocationQuota quotaResponse = await batchManagementClient.Location.GetQuotasAsync(location);
```
Comment (cecilphillip) [inline api/overview/azure/batch.md]: BatchLocationQuota should be BatchLocationQuotaInner based on the currently released NuGet package

File: api/overview/azure/batch.md [hunk 6]
```diff
@@ -47,10 +87,40 @@ Install-Package Microsoft.Azure.Management.Batch.Fluent
...
+    Console.WriteLine("Your subscription can create {0} account(s) in the {1} region.", quotaResponse.AccountQuota, location);
+
+    // Create account
+    await batchManagementClient.BatchAccount.CreateAsync(ResourceGroupName, accountName, new BatchAccountCreateParameters() { Location = location });
```
Comment (cecilphillip) [inline api/overview/azure/batch.md]: BatchAccountCreateParameters should be BatchAccountCreateParametersInner

File: api/overview/azure/batch.md (hunk 1 above)
Comment (JeremyLikness) [inline api/overview/azure/batch.md]: The bash is just to get the highlighting, it doesn't appear in the output

**Comment by JeremyLikness** [comment]
@cecilphillip I made updates per your feedback, please review second pass. Thanks!

**Comment by openpublishbuild** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api/overview/azure/index.yml](https://github.com/JeremyLikness/azure-docs-sdk-dotnet/blob/patch-5/api/overview/azure/index.yml) | :x:Error | | [Details](#user-content-api%2Foverview%2Fazure%2Findex.yml)
[]() | :x:Error | | [Details](#user-content-)

<a name="api%2Foverview%2Fazure%2Findex.yml"></a>
### [api/overview/azure/index.yml](https://github.com/JeremyLikness/azure-docs-sdk-dotnet/blob/patch-5/api/overview/azure/index.yml)
 - **[Error]** Error transforming model &quot;W:\ppan-o-f9765a29\raw-\zkwttukr.51t\rawmodel\api\overview\azure\index.raw.json&quot; generated from &quot;api/overview/azure/index.yml&quot; using &quot;ManagedReference.html.primary.js&quot;. Error running Transform function inside template preprocessor: 

<a name=""></a>
### []()
 - **[Error]** Error transforming model &quot;W:\ppan-o-f9765a29\raw-\zkwttukr.51t\rawmodel\api\overview\azure\index.raw.json&quot; generated from &quot;api/overview/azure/index.yml&quot; using &quot;ManagedReference.html.primary.js&quot;. Error running Transform function inside template preprocessor: 

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C8%5C1%5C14458122-e42f-10e3-bd3d-ee1c7494cfbf%5CPullRequest%5C201708011706267944-276%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api/overview/azure/batch.md (hunk 1 above)
Comment (cecilphillip) [inline api/overview/azure/batch.md]: Ok. Could you add "NET Core command line" over this command; similar to how it is [here](https://docs.microsoft.com/en-us/dotnet/api/overview/azure/sql?view=azure-dotnet)

**Comment by JeremyLikness** [comment]
I think we've got it now @cecilphillip - please sign-off or let me know if there's anything else, thanks!

**Comment by openpublishbuild** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api/overview/azure/index.yml](https://github.com/JeremyLikness/azure-docs-sdk-dotnet/blob/patch-5/api/overview/azure/index.yml) | :x:Error | | [Details](#user-content-api%2Foverview%2Fazure%2Findex.yml)
[]() | :x:Error | | [Details](#user-content-)

<a name="api%2Foverview%2Fazure%2Findex.yml"></a>
### [api/overview/azure/index.yml](https://github.com/JeremyLikness/azure-docs-sdk-dotnet/blob/patch-5/api/overview/azure/index.yml)
 - **[Error]** Error transforming model &quot;W:\lxga-o-4979efa6\raw-\n0j5m0gu.3ls\rawmodel\api\overview\azure\index.raw.json&quot; generated from &quot;api/overview/azure/index.yml&quot; using &quot;ManagedReference.html.primary.js&quot;. Error running Transform function inside template preprocessor: 

<a name=""></a>
### []()
 - **[Error]** Error transforming model &quot;W:\lxga-o-4979efa6\raw-\n0j5m0gu.3ls\rawmodel\api\overview\azure\index.raw.json&quot; generated from &quot;api/overview/azure/index.yml&quot; using &quot;ManagedReference.html.primary.js&quot;. Error running Transform function inside template preprocessor: 

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C8%5C1%5C14458122-e42f-10e3-bd3d-ee1c7494cfbf%5CPullRequest%5C201708011910199839-276%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by cecilphillip** [comment]
#sign-off

**Comment by openpublishbuild** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api/overview/azure/index.yml](https://github.com/JeremyLikness/azure-docs-sdk-dotnet/blob/patch-5/api/overview/azure/index.yml) | :x:Error | | [Details](#user-content-api%2Foverview%2Fazure%2Findex.yml)
[]() | :x:Error | | [Details](#user-content-)

<a name="api%2Foverview%2Fazure%2Findex.yml"></a>
### [api/overview/azure/index.yml](https://github.com/JeremyLikness/azure-docs-sdk-dotnet/blob/patch-5/api/overview/azure/index.yml)
 - **[Error]** Error transforming model &quot;W:\c4kw-o-aeb1350a\raw-\sdndzko0.unq\rawmodel\api\overview\azure\index.raw.json&quot; generated from &quot;api/overview/azure/index.yml&quot; using &quot;ManagedReference.html.primary.js&quot;. Error running Transform function inside template preprocessor: 

<a name=""></a>
### []()
 - **[Error]** Error transforming model &quot;W:\c4kw-o-aeb1350a\raw-\sdndzko0.unq\rawmodel\api\overview\azure\index.raw.json&quot; generated from &quot;api/overview/azure/index.yml&quot; using &quot;ManagedReference.html.primary.js&quot;. Error running Transform function inside template preprocessor: 

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C8%5C1%5C14458122-e42f-10e3-bd3d-ee1c7494cfbf%5CPullRequest%5C201708011913048645-276%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**